app/views/account/comments/_form.html.slim

```diff
@@ -1,5 +1,5 @@
 .container.box
   = simple_form_for [:account, commentable, comment], remote: true do |f|
     = f.input :body, placeholder: 'Add a comment...', label: 'New comment',
-              wrapper: :textarea, required: true
+              wrapper: :textarea, required: true, as: :text
```

Comment:
pls follow the pattern:
`f.input <attribute>, <as: [input]>, <*options>, <*html_options>`